event/callback.go

```diff
@@ -0,0 +1,66 @@
+package event
```

Comment:
Is this file the same as dp-api-clients-go/errors.callback.go? Maybe you can use the errors package? (although this is under event...)